nifi/Dockerfile

```diff
@@ -4,6 +4,65 @@
 # Ignoring DL4006 globally because we inherit the SHELL from our base image
 # hadolint global ignore=DL3038,DL4006
 
+FROM stackable/image/java-devel AS builder
```

Comment:
Could you add a hadolint ignore rule for this?
Look at Kafka for an example.

Reply:
https://github.com/stackabletech/docker-images/pull/678/commits/4cbb2b52e56632e3e64dfa645490e97b9b1faf4e